docs/templates/language-reference.md

```diff
@@ -49,10 +49,30 @@ Depending on the language, a brief description of the recommended way to handle
 If available, include examples for connecting in an app
 (taken from https://examples.docs.platform.sh/).
 
-### Configuration reader
+{{< codetabs >}}
 
-If available, include the configuration reader available for the given language
-(for example, for [Python](https://github.com/platformsh/config-reader-python)).
+---
+title=<EXAMPLE_1>
+file=static/files/fetch/examples/<LANGUAGE_TYPE>/<SERVICE>
```

Comment:
:lady_beetle: I'd be consistent here. If "TYPE" is to indicate it should be the machine name, then it should be for both.

Reply:
This wasn't resolved. Note that `<SERVICE>` in line 57 is different than `<SERVICE>` in line 58

Reply:
Hmm maybe it would be clearer when removing the type altogether?
Or would it be lacking the differences with capitalization + the slightly different name for the highlight?

Reply:
I would personally be inclined for the suggestion below to distinguish between the human-readable name and the machine-readable name.